### PR TITLE
feat(i18n-it)

### PR DESCRIPTION
**Pull request by RazvanAton**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/i18n/languages/italian/addon-editor.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        subscript: 'Pedice',
+        insertTable: 'Inserisci tabella',
+        mergeCells: 'Unisci celle',
+        splitCells: 'Dividi celle',
+        rowsColumnsManaging: 'Gestione righe e colonne',
+        cellColor: 'Colore cella',
+    },
+    editorTableCommands: [
+        ['Inserisci colonna prima', 'Inserisci colonna dopo'],
+        ['Inserisci colonna prima', 'Inserisci colonna dopo'],
```
Comment (nsbarsukov) [inline projects/i18n/languages/italian/addon-editor.ts:46]: Hi

Thanks for your PR!
I don't know italian but I suppose that there is an error here. 
In English this row:
```
['Insert row before', 'Insert row after'],
```

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/i18n/languages/italian/addon-editor.ts (hunk 1 above)
Comment (RazvanAton) [reply projects/i18n/languages/italian/addon-editor.ts:46]: Hi @nsbarsukov,
My bad! I'll fix it immediately! 
Thank you!

**Comment by nsbarsukov** [comment]
@RazvanAton add new language in this [README](https://github.com/TinkoffCreditSystems/taiga-ui/blob/main/projects/i18n/README.md), please

**Review by nsbarsukov — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

File: projects/i18n/languages/italian/kit.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    mobileCalendarTexts: ['Scegli giorno', 'Scegli intervallo'],
+    range: ['da', 'a'],
+    countTexts: ['Più', 'Meno'],
+    time: {
+        'HH:MM': 'HH:MM',
+        'HH:MM:SS': 'HH:MM:SS',
+        'HH:MM:SS.MSS': 'HH:MM:SS.MSS',
+    },
+    dateTexts: {
+        DMY: 'gg.mm.aaa',
```
Comment (nsbarsukov) [inline projects/i18n/languages/italian/kit.ts:17]: @RazvanAton why only 3 letters `a` ?

Comment (RazvanAton) [reply projects/i18n/languages/italian/kit.ts]: @nsbarsukov sorry! my bad again..

**Review by nsbarsukov — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**